composer/core/time.py

```diff
@@ -379,6 +383,9 @@ def __init__(
         batch_in_epoch: Union[int, Time[int]] = 0,
         sample_in_epoch: Union[int, Time[int]] = 0,
         token_in_epoch: Union[int, Time[int]] = 0,
+        total_duration: Optional[datetime.timedelta] = None,
```

Comment:
Thoughts on renaming `{total, epoch, batch}_duration` to `{total, epoch, batch}_wct_duration`? It is a bit more clear what it does, especially since we mess with the notion of `Time` a lot.

Reply:
Done